.circleci/config.yml

```diff
@@ -203,12 +215,7 @@ workflows:
         context: AWS_ECR_CREDENTIALS_PSGA
     - build-sars-cov-2-pipeline:
         requires:
-          - build-read-it-and-keep
-          - build-fastqc
-          - build-primer-autodetection
-          - build-ncov2019-artic-nf-illumina
-          - build-ncov2019-artic-nf-nanopore
-          - build-pangolin
```

Comment:
This was relevant when running Jenkins integration tests triggered on the PR branch.